hs/MasterTemplate.hs

```diff
@@ -23,15 +23,16 @@ masterTemplate title headers body scripts =
 -- Insert the header of the Grid and Graph. This contains the year of the timetable, and
 -- a link back to the Graph.
 header :: String -> H.Html
-header page = createTag H.nav "" "row header" $ do
-  H.h2 "Courseography"
-  H.ul ! A.id "nav-links" $ do
-    H.li $ makeA "" "" "graph" "" $ "Graph"
-    H.li $ makeA "" "" "grid" "" $ "Grid"
-    H.li $ makeA "" "" "draw" "" $ "Draw"
-    H.li $ makeA "" "" "post" "" $ "Check My POSt!"
-    H.li $ makeA "" "" "about" "" $ "About"
-
+header page = 
+  createTag H.nav "" "row header" $ do
```

Comment:
This function is off.

1. Make the indentation 4 spaces.
2. What didn't you remove the `$` for "draw"?